Sources/EosioSwift/EosioRpcProvider/Endpoints/EosioRpcProviderEndpointsCallbacks.swift

```diff
@@ -28,7 +28,7 @@ extension EosioRpcProvider {
     /// - Parameters:
     ///   - requestParameters: An `EosioRpcBlockRequest`.
     ///   - completion: Called with the response, as an `EosioResult` consisting of an `EosioRpcBlockResponse` and an optional `EosioError`.
-    public func getBlock(requestParameters: EosioRpcBlockRequest, completion: @escaping (EosioResult<EosioRpcBlockResponse, EosioError>) -> Void) {
+    public func _getBlock(requestParameters: EosioRpcBlockRequest, completion: @escaping (EosioResult<EosioRpcBlockResponse, EosioError>) -> Void) {
```

Comment:
I still don't think this is the way to go. Prefixing with an underscore is a sign that people shouldn't use these directly. But there are many use cases for people to call `getBlock` or `getInfo` from their apps directly. The versions of these that return the protocol types, however, are not meant for external consumption. It'd be better, IMHO, to prefix those instead. I realize that's more intrusive with code changes. But I think it's the right thing to do for the RPC API. Thoughts?

Reply:
Well, not really since you need to expose the protocol returning versions in EosioRpcProviderProtocol or you'll bind everyone to our response objects rather than the protocol conforming ones.

Reply:
We can come up with something different than the underscore if that seems too charged but since I couldn't think of a good name, I just went with that for now.

Reply:
@brandonfancher Updated to change the `EosioRpcProviderProtocol` and those conforming implementations to add the `Base` suffix as discussed.